lang/cs/Org.Apache.REEF.Network/Naming/Events/NamingEvent.cs

```diff
@@ -25,7 +25,7 @@ namespace Org.Apache.REEF.Network.Naming.Events
     /// Event representing a lookup, registering, or unregistering of 
     /// an identifier with the Name Service.
     /// </summary>
-    internal class NamingEvent
+    internal abstract class NamingEvent
```

Comment:
OK. In general, I am not a fan of behavior inheritance, but we shouldn't change it in this PR. `abstract` it is.